source/renderer/app/utils/transaction.js

```diff
@@ -0,0 +1,275 @@
+// @flow
+import React from 'react';
+import moment from 'moment';
+import BigNumber from 'bignumber.js';
+import {
+  WalletTransaction,
+  TransactionTypes,
+} from '../domains/WalletTransaction';
+import { formattedWalletAmount } from './formatters';
+import type {
+  DateRangeType,
+  TransactionFilterOptionsType,
+} from '../stores/TransactionsStore';
+import { DateRangeTypes } from '../stores/TransactionsStore';
+
+const AMOUNT_RAW_LENGTH_LIMIT = 10;
```

Comment:
@yakovkaravelov is this still used?

Reply:
@nikolaglumac [yes](https://github.com/input-output-hk/daedalus/pull/1815/files/ced088a18ce9d97a84ca976f25cf4701c85d6551#diff-077912e23fd9bb7a988d93509c8d13f1R243)